src/sub/tests/unit/test_subhub.py

```diff
@@ -366,3 +387,420 @@ def test_plan_response_invalid(mock_plans, app):
     )
 
     assert response.status_code == 500
+
+
+class PatchCustomerSubscriptionTest(unittest.TestCase):
+    """
+    Tests the upgrade/downgrade subscription endpoint
+    """
+
+    def setUp(self) -> None:
+        sub_app = create_app()
+        with sub_app.app.app_context():
+            g.subhub_account = sub_app.app.subhub_account
+            g.subhub_deleted_users = sub_app.app.subhub_deleted_users
+
+        self.client = sub_app.app.test_client()
+
+        with open(
+            os.path.join(os.path.dirname(__file__), "fixtures/stripe_cust_test1.json")
```

Comment:
these have the `with` expressions; strange why the others do not; consistency for the win;  as before consider calculating the dirname of __file__ once and then use it in each; should keep these ugly black wraps from happening; also note that with file handles like this, it is very common to see `f` used not `fh`